.unreleased/LLT-5023

```diff
@@ -0,0 +1 @@
+Add no-link enhanced detection mod
```

Comment:
Some time ago we talked about this feature being "Link detection" if you still agree to that, could you rename all `no-link` parts to `Link detection` so it would be less ambiguous? :)

Reply:
You're right, I forgot to update it.